src/main/java/com/commercetools/pingpong/model/Player.java

```diff
@@ -21,17 +23,30 @@ public Player(final String name) {
     public Player() {
     }
 
-    public void add() {
+    public void addPoint() {
         setScore++;
     }
 
-    public void sub() {
+    public void subPoint() {
         setScore--;
     }
 
-    public void reset() {
+    public void resetSet() {
         setScore = 0;
-        matchScore = 0;
+    }
+
+    public void setBeginningServe() {
+        beginningServe = true;}
+
+    public void changeBeginningServe() {
+        beginningServe = false;}
+
+    public void setServe() {
+        serve = true;
+    }
+
+    public void changeServe() {
```

Comment:
If `setServe` sets the serve to true, then setting the serve to `false` would make more sense via a `unsetServe` method.

Reply:
Yes, that's better
76c872d8d0a425144640f13d4377bbdcd6e182c0